### ISSUE TITLE
Parcel with Vue js - watch not working

### ISSUE THREAD

**Issue by icrismariu** [parcel-bundler/parcel#2552]
# 🐛 bug report

Using watch, if i do changes in app.js file or main vue file imported into app.js it works, if i'm doing changes in other .vue files imported in main .vue file it doesn't work.

## 🎛 Configuration (.babelrc, package.json, cli command)

package.json
`"watch": "parcel watch --out-dir static/default/js app/app.js"`

## 🤔 Expected Behavior

Changing a .vue file imported in main app.vue - parcel watch should recompile

## 😯 Current Behavior

It works only for changes done in main .vue file

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |  1.11.0
| Node             | v10.10.0
| npm/Yarn         | 6.4.1
| Operating System | macOS Mojave

**Comment by icrismariu** [comment]
TLDR: be aware of letter casing

My mistake, it's not a bug, but a "casing" issue. 

The file name was "SomeComponent.vue" and it was Imported as "somecomponent.vue" 

Fixed the casing and now it works correctly!
